src/main/java/life/qbic/presenter/MainPresenter.java

```diff
@@ -80,14 +84,59 @@ private void addOrganismCountPie(){
 
     private void addWorkflowGrid(){
 
-        WorkflowPresenter workflowPresenter = new WorkflowPresenter(mainConfig.getCharts().get(ChartNames.Workflow.toString()), mainView);
+        Map<String, ChartConfig> availWorkflows = new HashMap<>();
+        mainConfig.getCharts().keySet().forEach(key ->{
+            if(key.startsWith(ChartNames.Available_Workflows_.toString())){
+                availWorkflows.put(key, mainConfig.getCharts().get(key));
+            }
+        });
+        WorkflowPresenter workflowPresenter = new WorkflowPresenter(mainConfig.getCharts()
+                .get(ChartNames.Workflow_Execution_Counts.toString()),
+                mainView,
+                availWorkflows);
         workflowPresenter.specifyView(new TabView(workflowPresenter.getView(), workflowPresenter.getModel()),
-                                    "Workflows");
+                "Workflows");
+
+        addReturnButtonListener(workflowPresenter.getTabView());
 
         logger.info("Workflow tab was added");
 
     }
 
+    private void addSampleCountPie(){
+
```

Comment:
Just for the sake of consistency: Do we added new lines after method definitions or not? 
I personally prefer not to add new lines, but I don't mind either way ;)